File: C/tacografo.c
#include <stdio.h>

int main()
{
    int n, total, a, b, i;
    scanf("%d", &n);
    total = 0;
    for (i = 0; i < n; i++)
    {
        scanf("%d %d", &a, &b);
        total = total + (a * b);
    }

    printf("%d\n", total);

    return 0;
}

File: dart/tacografo.dart
import 'dart:io';

void main(List<String> args) {
  int a = int.parse(stdin.readLineSync()!);
  int total = 0;
  for (int i = 0; i < a; i++) {
    var vetor = stdin.readLineSync()!.split(' ').map(int.parse).toList();
    total = total + (vetor[0] * vetor[1]);
  }

  print(total);
}
